### ISSUE TITLE
Cannot choose to join any team besides red or blue

### ISSUE THREAD

**Issue by WolfieWaffle** [alkarinv/BattleArena#310]
I made an arena with four teams, and tried to do /a join arenaname blue. It worked.
I tried /a join arenaname green, and it did not work. It gave me this error: pastebin.com/E4mr2Rwi

I think it has to do with the competition size being only 2, and it trying to make me join a 3rd team. But for that individual arena I have it set to 2-4 teams.

Sorry if I didn't provide enough information, I'm new to GitHub and issues and stuff.

**Comment by Europia79** [comment]
> I think it has to do with the competition size being only 2, and it trying to make me join a 3rd team. But for that individual arena I have it set to 2-4 teams.

As you can see, in the error log, it shows `numTeams=2`

```
[13:37:23] [Server thread/WARN]: [Error] object=Paintball:Paintball,numTeams=2,teamSize=1+ options=
```

Can you show me your `arenas.yml` and the corresponding `/BattleArena/competitions/` Config.yml for the arena called `4square`

It appears to be of type Paintball ? If so, the files will be located in `/plugins/ArenaPaintball/arenas.yml` and `PaintballConfig.yml`

**Comment by WolfieWaffle** [comment]
OK, here they are:
arenas: pastebin.com/y5B8hKNx
PaintballConfig: pastebin.com/WADH9weu

Since I posted this, I made the default for the competition 2-4 teams, but still gives me the same error.

**Comment by Europia79** [comment]
Each Paintball arena will inherit settings from PaintballConfig.yml

But you can override the parent settings by changing the `params:` section of `arenas.yml`. Which is what you have done: Inside `arenas.yml` you have 

```
  params:
    gameSize:
      nTeams: '2'
```

You need to either update this number, or delete the `params:` section... So that you'll inherit `2-4` from `PaintballConfig.yml`

**Comment by WolfieWaffle** [comment]
yes, but I have no params on the arena that I'm trying to join as green.
I'm trying to join /pb join 4square green and it gives me the error. It
should already be inheriting 2-4.

**Comment by Europia79** [comment]
Oh, i see... I mis-read your `arenas.yml`. The `params:` section is for the other arena.

What exactly are you wanting to do ? 

Are you wanting the ability to choose your team color ? If so, I don't think that BattleArena currently offers this.

Are you wanting to join the 3rd team ? If so, that's white

`/plugins/BattleArena/teamConfig.yml` has all the team colors. By default, team3 is white. team4 is black.

I think you can change `teamConfig.yml` to make green the 3rd team (if that's what you want).

**Comment by WolfieWaffle** [comment]
Yes, I am trying to join the third team (which I have configured to be green) on the 4square arena, which allows 2-4 teams :). My problem is that when I try to do this, it breaks or something and doesn't let me join the third team (green). It gives me that out of bounds error, so it must not be registering the 3rd team. I'm wondering if this is my problem or a problem with the plugin.
